src/os/bluestore/bluestore_types.h

```diff
@@ -272,6 +272,121 @@ static inline bool operator!=(const bluestore_extent_ref_map_t& l,
   return !(l == r);
 }
 
+/// blob_use_tracker: a set of per-alloc unit ref counters to track blob usage
+struct bluestore_blob_use_tracker_t {
+  // N.B.: There is no need to minimize ref_size/refs_count
+  //   as much as possible (e.g. have just a single byte for ref_size) since:
+  //   1) Struct isn't packed hence it's padded. And even if it's packed see 2)
+  //   2) Mem manager has its own granularity, most probably >= 8 bytes
+  //
+  uint32_t ref_size;
+  uint32_t refs_count; 
+                       
+  union {
+    uint32_t* ref_by_au;
+    uint32_t ref_total;
+  };
```

Comment:
Having a hard time following what these fields mean.

I think refs_count is more like num_au?